pkg/crypto/crypto_test.go

```diff
@@ -0,0 +1,358 @@
+package crypto
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"github.com/mr-tron/base58/base58"
+	"github.com/stretchr/testify/assert"
```

Comment:
Let's use the standart test tooling for now to make our code and tests look the same way